pom.xml

```diff
@@ -12,9 +12,19 @@
     <dependencyManagement>
         <dependencies>
             <dependency>
+                <groupId>com.google.inject</groupId>
+                <artifactId>guice</artifactId>
+                <version>4.0</version>
+            </dependency>
+            <dependency>
+                <groupId>com.google.inject.extensions</groupId>
+                <artifactId>guice-multibindings</artifactId>
```

Comment:
Why do we use 4.0? Spoofax uses 3.0. I am not sure if a mixed installation is a wise idea.